### ISSUE TITLE
Show SST internal configuration when verbose

### ISSUE THREAD

**Issue by pdbj** [sstsimulator/sst-core#838]
It's often useful to document the internal SST configuration.  This patch logs Config to the default simulator output, when verbose.

Patch [Show SST configuration when verbose](https://github.com/pdbj/sst-core/commit/d3e86c13b6b365b3ce4d6bc18784c130a4428f96)

**Comment by pdbj** [comment]
Rebased [Show SST configuration when verbose](https://github.com/pdbj/sst-core/commit/b4f3de97beff020df6bc537624cebcacb7ec3635)

**Comment by jleidel** [comment]
@feldergast, I can prep an issue-fix branch for a forthcoming PR if you approve.

**Comment by feldergast** [comment]
This isn't the way I would fix this issue.  If there are multiple ways to print (Output object versus stdout, for example), I've been trying to consolidate things into a toString() function and then using that in the various print functions.  Also, [[fallthrough]] is c++17, so isn't allowed in the core.

**Comment by pdbj** [comment]
Yes, I wasn't wild about the duplication.  On the other hand my new function prints things with a little more formatting (column aligned), so some reconciling needs to occur.

**Comment by feldergast** [comment]
I have some changes to the config class on a branch I'm working on.  I'll take the new code and tweak it into a toString() function.  I'll use that in print() versions to take FILE* and Output.  There's probably not a great reason to have a version that only goes to stdout.
